features/tasks/nuxtSubmitTask.feature

```diff
@@ -4,7 +4,7 @@ Feature: View the task in new nuxt task menu and submit it
     Background: User submits the opened tasks and view the tasks in completed tab
         Given user arrives on the Schul-Cloud homepage
 
-    @nuxtSubmitAndViewTask
+    @nuxtSubmitAndViewTask @unstableTest
```

Comment:
Please add this test to https://docs.hpi-schul-cloud.org/pages/viewpage.action?pageId=149292278 and create a ticket to fix it.